api/nakadi-event-bus-api.yaml

```diff
@@ -925,6 +1199,67 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: |
+      Subscription is a high level consumption unit. Subscriptions allow applications to easily scale the
+      number of clients by managing consumed event offsets and distributing load between instances.
+      The key properties that identify subscription are 'owning_application', 'event_types' and 'consumer_group'.
+      It's not possible to have two different subscriptions with these properties being the same.
+    properties:
+      id:
+        type: string
+        format: uuid
+        readOnly: true
+        description: |
+          Id of subscription that was created. Is generated by Nakadi, should not be specified when creating
+          subscription.
+      owning_application:
+        type: string
+        example: 'gizig'
+        description: |
+          The id of application owning the subscription. The reading and modification of subscription will be limited
+          to this application id.
+      event_types:
+        type: array
+        items:
+          type: string
+        description: |
+          EventTypes to subscribe to.
+          The order is not important. Subscriptions that differ only be the order of EventTypes will be
+          considered the same and will have the same id.
+
+          * Currently only subscription to a single EventType is supported. Subscriptions with more than one
+            EventType in event_types property will be rejected.
+      consumer_group:
+        type: string
+        example: 'read-product-updates'
+        description: |
+          The value describing the use case of this subscription.
+          In general that is an additional identifier used to differ subscriptions having the same
+          owning_application and event_types.
+        default: 'default'
+      created_at:
+        type: string
+        readOnly: true
+        description: |
+          Timestamp of creation of the subscription. Is generated by Nakadi, should not be specified when
+          creating subscription.
```

Comment:
I think it will result in error

👍 

```
Timestamp of creation of the subscription. Is generated by Nakadi, should not be specified when creating subscription and sending it may result in a client error.
```